File: lib-segment/src/test/java/websiteschema/mpsegment/SimpleDomainDictLoader.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package websiteschema.mpsegment;

import websiteschema.mpsegment.dict.DictionaryException;
import websiteschema.mpsegment.dict.domain.DomainDictLoader;
import websiteschema.mpsegment.dict.domain.DomainDictionary;

public class SimpleDomainDictLoader implements DomainDictLoader {

    @Override
    public void load(DomainDictionary dict) {
        try {
            dict.pushWord("PC机", null, "N", 5, 10001);
            dict.pushWord("个人电脑", "PC机", "N", 5, 10001);
        } catch (DictionaryException e) {
            e.printStackTrace();
        }
    }
}

File: lib-segment-scala/src/test/scala/websiteschema/mpsegment/dict/SimpleDomainDictLoader.scala
package websiteschema.mpsegment.dict

import websiteschema.mpsegment.dict.domain.DomainDictLoader
import websiteschema.mpsegment.dict.domain.DomainDictionary

class SimpleDomainDictLoader extends DomainDictLoader {

  override def load(dict: DomainDictionary) {
    try {
      dict.pushWord("PC机", null, "N", 5, 10001);
      dict.pushWord("个人电脑", "PC机", "N", 5, 10001);
    } catch {
      case e: Throwable =>
        e.printStackTrace();
    }
  }
}
